### ISSUE TITLE
[REVIEW]: gospl: Global Scalable Paleo Landscape Evolution

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2804]
**Submitting author:** @tristan-salles (<a href="http://orcid.org/0000-0001-6095-7689">Tristan Salles</a>)
**Repository:** <a href="https://github.com/Geodels/gospl/" target ="_blank">https://github.com/Geodels/gospl/</a>
**Version:** v0.2.1
**Editor:** @kbarnhart
**Reviewer:** @johnjarmitage, @cmshobe
**Archive:** <a href="https://doi.org/10.5281/zenodo.4319332" target="_blank">10.5281/zenodo.4319332</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/458fab1e8ad452f66400080617d8cfed/status.svg)](https://joss.theoj.org/papers/458fab1e8ad452f66400080617d8cfed)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/458fab1e8ad452f66400080617d8cfed"><img src="https://joss.theoj.org/papers/458fab1e8ad452f66400080617d8cfed/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/458fab1e8ad452f66400080617d8cfed/status.svg)](https://joss.theoj.org/papers/458fab1e8ad452f66400080617d8cfed)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/Geodels/gospl/" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@johnjarmitage & @cmshobe, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @kbarnhart know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/Geodels/gospl/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@tristan-salles) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/Geodels/gospl/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@tristan-salles) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @johnjarmitage, @cmshobe it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: :point_left:

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1016/j.sedgeo.2018.06.007 is OK
- 10.2110/jsr.2016.56 is OK
- 10.1002/2013WR014326 is OK
- 10.1016/j.cageo.2013.04.024 is OK
- 10.1016/j.epsl.2019.115728 is OK
- 10.5194/gmd-12-4165-2019 is OK
- 10.1016/j.earscirev.2015.10.013 is OK
- 10.1016/j.geomorph.2015.04.007 is OK
- 10.1371/journal.pone.0195557 is OK
- 10.1016/j.margeo.2011.02.014 is OK

MISSING DOIs

- 10.1046/j.1365-2117.1997.00030.x may be a valid DOI for title: Modelling landscape evolution on geological time scales: a new method based on irregular spatial discretization
- 10.1007/978-1-4615-0575-4_12 may be a valid DOI for title: The Channel-Hillslope Integrated Landscape Development Model (CHILD)
- 10.5194/esurf-5-21-2017 may be a valid DOI for title: Creative computing with Landlab: an open-source toolkit for building, coupling, and exploring two-dimensional numerical models of Earth-surface dynamics
- 10.1016/j.geomorph.2012.10.008 may be a valid DOI for title: A very efficient O(n), implicit and parallel method to solve the stream power equation governing fluvial incision and landscape evolution
- 10.2110/pec.99.62.0197 may be a valid DOI for title: Concepts and applications of a 3D multiple lithology, diffusive model in stratigraphic modeling.
- 10.21105/joss.00964 may be a valid DOI for title: eSCAPE: parallel global-scale landscape evolution model
- 10.1029/2010jf001935 may be a valid DOI for title: Field calibration of sediment flux dependent river incision
- 10.1063/1.4823180 may be a valid DOI for title: Software for Portable Scientific Data Management

INVALID DOIs

- None
```

**Comment by kbarnhart** [comment]
@johnjarmitage @cmshobe thanks for agreeing to review this submission to JOSS. As noted above, we are trying to adjust to authors, reviewers, and editors needs under COVID. Thus we are asking reviewers try and complete reviews in 6 weeks. I am going to use whedon the editorial bot to remind you about your review in two weeks, and then again in 4 weeks.

If there are any questions from the authors or reviewers throughout the process, please feel free to let me know here or directly at krbarnhart@usgs.gov. 

Thanks again for contributing to the JOSS review process.

**Comment by kbarnhart** [comment]
@whedon remind @johnjarmitage in 2 weeks

**Comment by whedon** [comment]
Reminder set for @johnjarmitage in 2 weeks

**Comment by kbarnhart** [comment]
@whedon remind @cmshobe in 2 weeks

**Comment by whedon** [comment]
Reminder set for @cmshobe in 2 weeks

**Comment by kbarnhart** [comment]
@tristan-salles could you add the DOI's identified as missing by whedon? There are also some parts of the bibliography (mostly titles) that could benefit from extra curly braces to enforce capitalization (useful reference [here](https://www.ece.ucdavis.edu/~jowens/biberrors.html)).

**Comment by tristan-salles** [comment]
@kbarnhart I have updated the DOI's and the Titles capitalisation in the bibliography.
Thank you to the reviewers

**Comment by whedon** [comment]
:wave: @johnjarmitage, please update us on how your review is going.

**Comment by whedon** [comment]
:wave: @cmshobe, please update us on how your review is going.

**Comment by kbarnhart** [comment]
@johnjarmitage @cmshobe. Please disregard that message. I expected it to occur two weeks from when I set it (one third of the way into the review). Not sure why it occurred now (FYI @arfon)

**Comment by kbarnhart** [comment]
@whedon remind @johnjarmitage in 1 week

**Comment by whedon** [comment]
Reminder set for @johnjarmitage in 1 week

**Comment by kbarnhart** [comment]
@whedon remind @cmshobe in 1 week

**Comment by whedon** [comment]
Reminder set for @cmshobe in 1 week

**Comment by kbarnhart** [comment]
@whedon generate pdf

**Comment by kbarnhart** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1016/j.sedgeo.2018.06.007 is OK
- 10.2110/jsr.2016.56 is OK
- 10.1002/2013WR014326 is OK
- 10.1016/j.cageo.2013.04.024 is OK
- 10.1016/j.epsl.2019.115728 is OK
- 10.5194/gmd-12-4165-2019 is OK
- 10.1046/j.1365-2117.1997.00030.x is OK
- 10.1007/978-1-4615-0575-4_12 is OK
- 10.5194/esurf-5-21-2017 is OK
- 10.1016/j.geomorph.2012.10.008 is OK
- 10.1002/esp.1952 is OK
- 10.2110/pec.99.62.0197 is OK
- 10.1016/j.earscirev.2015.10.013 is OK
- 10.1016/j.geomorph.2015.04.007 is OK
- 10.1371/journal.pone.0195557 is OK
- 10.21105/joss.00964 is OK
- 10.1029/2010jf001935 is OK
- 10.1063/1.4823180 is OK
- 10.1016/j.margeo.2011.02.014 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: :point_left:

**Comment by kbarnhart** [comment]
@johnjarmitage, @cmshobe I wanted to write with a quick check-in to see if you had any questions for me about the JOSS review process.

**Comment by johnjarmitage** [comment]
@kbarnhart yes: 
1. What level of explanation of the equations/methods being solved within the software package is expected? I have tested one aspect of the code and it works as it should. I am however not sure exactly what is going on with the lateral movement of the tectonic plates. In a science journal this might be a problem, but from the check list above I don't see this being flagged as an issue.
2. (Related) If I have suggestions for improving the description, where do I put them? An issue in the code repository, or is there a formal review in JOSS?

**Comment by kbarnhart** [comment]
@johnjarmitage great questions. 

_Regarding the first_: The two most relevant elements of the review criteria here would be:

> Functionality documentation: Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
> Automated tests: Are there automated tests or manual steps described so that the functionality of the software can be verified?

I would expect that the functional form of the equations being solved by the package are stated, their constants listed, a description of any boundary conditions, and a statement of numerical methods used, if any (e.g., solved with finite difference). I would expect that, if known solutions to the governing equations exist that some sort of test would exist to demonstrate the equations are being solved correctly. I would not expect that every equation be written out in a discretized form. 

This information could be located anywhere in the package documentation (e.g., narrative documentation, examples, api). 

If I understand your question correctly, I think you are indicating that may not be sufficient detail in describing how the tectonic boundary conditions are implemented on the discretized grid (e.g., warping of existing grid cells due to tectonic advection, creating or removing grid cells if they get too big or too small, what quantities are conserved). I would expect some description of how this is implemented and if there are any inputs exposed to the user that control this behavior. 

Let me know if this answers your question.

_Regarding the second_: We recommend that you make most recommendations/requests for change as issues in the source repository and link them here (just paste "openjournals/joss-reviews/issues/2804" into the new issue and github will do the linking). Thus this issue compiles the major topics of discussion, requested changes that are motivated by this review, but does not contain them.

**Comment by johnjarmitage** [comment]
@kbarnhart, I think I did it right. Thanks!

**Comment by kbarnhart** [comment]
@johnjarmitage looks great to me. Now @tristan-salles can respond to your common on the new issue Geodels/gospl#94 and if discussion is needed it can happen there.

**Comment by tristan-salles** [comment]
Thanks @johnjarmitage. Following your comments I have updated the technical guidelines. If you are happy with the changes I will close the open issue on GitHub (@kbarnhart).

**Comment by johnjarmitage** [comment]
@tristan-salles I closed the issue, as I thought the technical note was good enough. While I am here, I wonder if the JOSS article should mention the divergence between gospl, pyBadlands (or just Badlands) and eSCAPE. I think all of these codes use a similar approach, therefore why do we need gospl? What differentiates it from the others? Is it a simplified Badlands (no carbonates etc)?  Does eSCAPE only do global simulations?

**Comment by kbarnhart** [comment]
@whedon re-invite @cmshobe as reviewer

**Comment by whedon** [comment]
OK, the reviewer has been re-invited.

@cmshobe please accept the invite by clicking this link: https://github.com/openjournals/joss-reviews/invitations

**Comment by tristan-salles** [comment]
> @tristan-salles I closed the issue, as I thought the technical note was good enough. While I am here, I wonder if the JOSS article should mention the divergence between gospl, pyBadlands (or just Badlands) and eSCAPE. I think all of these codes use a similar approach, therefore why do we need gospl? What differentiates it from the others? Is it a simplified Badlands (no carbonates etc)? Does eSCAPE only do global simulations?

I have updated the paper accordingly, adding a new paragraph in the *Statement of need* section about the differences with these packages.

**Comment by johnjarmitage** [comment]
@tristan-salles excellent. Super clear to me now.

**Comment by tristan-salles** [comment]
Big thank you to @johnjarmitage and @cmshobe for agreeing to complete the review! I know time is precious at the moment. I think I have made the changes suggested by both reviewers.

**Comment by kbarnhart** [comment]
@johnjarmitage and @cmshobe - thanks for your reviews. 
@tristan-salles - thanks for addressing reviewer issues. 

Based on the discussion, it seems like there are no more reviewer comments. If this is the case, could both reviewers:
(a) look through their checklists (earlier on this thread) and check off completed boxes. If there are any remaining comments related to checking off an item, of course mention it so it can be addressed. 
(b) If all check boxes are checked, indicate that you are happy with the current state of the submission. 

Thanks!

**Comment by johnjarmitage** [comment]
I am happy with the submission. Its a great wee model that Tristan, Claire and Sabin developed. I hope it gets used.

**Comment by tristan-salles** [comment]
Thanks @johnjarmitage !

**Comment by cmshobe** [comment]
I am sorry for being late-- This is my first JOSS review and I had forgotten about the formal checklist! I am ready to see this published. This is a fantastic software package and I look forward to seeing where people go with it in the future.

**Comment by kbarnhart** [comment]
@johnjarmitage and @cmshobe thanks for your affirmation and your reviews. (no worries @cmshobe)

@tristan-salles at this point I'll do final copyediting checks and let you know the next steps. It may take a few days because of AGU.

**Comment by kbarnhart** [comment]
@whedon check references

**Comment by kbarnhart** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1016/j.sedgeo.2018.06.007 is OK
- 10.1029/94JB00744 is OK
- 10.1111/j.1365-2117.1992.tb00136.x is OK
- 10.2110/jsr.2016.56 is OK
- 10.1002/2013WR014326 is OK
- 10.1016/j.cageo.2013.04.024 is OK
- 10.1016/j.epsl.2019.115728 is OK
- 10.5194/gmd-12-4165-2019 is OK
- 10.1046/j.1365-2117.1997.00030.x is OK
- 10.1007/978-1-4615-0575-4_12 is OK
- 10.5194/esurf-5-21-2017 is OK
- 10.1016/j.geomorph.2012.10.008 is OK
- 10.1002/esp.1952 is OK
- 10.2110/pec.99.62.0197 is OK
- 10.1016/j.earscirev.2015.10.013 is OK
- 10.1016/j.geomorph.2015.04.007 is OK
- 10.1371/journal.pone.0195557 is OK
- 10.21105/joss.00964 is OK
- 10.1029/2010jf001935 is OK
- 10.1063/1.4823180 is OK
- 10.1016/j.margeo.2011.02.014 is OK

MISSING DOIs

- None

INVALID DOIs

- https://doi.org/10.1029/JB085iB07p03711 is INVALID because of 'https://doi.org/' prefix
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: :point_left:

**Comment by kbarnhart** [comment]
@tristan-salles here are the last steps:
- [ ] Merge PR I've opened with copyediting. Full disclosure, I've added a reference that is my own work, but this is because Landlab now has two core references.

Once that is done, please run `@whedon check references` if no invalid DOIs come up, please do the following:

- [ ] Make a tagged release of your software, and list the version tag of the archived version here.
- [ ] Archive the reviewed software in Zenodo or a similar service (e.g., figshare, an institutional repository)
- [ ] Check the archival deposit (e.g., in Zenodo) has the correct metadata. This includes the title (should match the paper title) and author list (make sure the list is correct and people who only made a small fix are not on it). You may also add the authors' ORCID.
- [ ] Please list the DOI of the archived version here.

I will then move forward with recommending the submission be accepted and published

**Comment by tristan-salles** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1016/j.sedgeo.2018.06.007 is OK
- 10.1029/JB085iB07p03711 is OK
- 10.1029/94JB00744 is OK
- 10.1111/j.1365-2117.1992.tb00136.x is OK
- 10.2110/jsr.2016.56 is OK
- 10.1002/2013WR014326 is OK
- 10.1016/j.cageo.2013.04.024 is OK
- 10.1016/j.epsl.2019.115728 is OK
- 10.5194/gmd-12-4165-2019 is OK
- 10.1046/j.1365-2117.1997.00030.x is OK
- 10.1007/978-1-4615-0575-4_12 is OK
- 10.5194/esurf-5-21-2017 is OK
- 10.5194/esurf-8-379-2020 is OK
- 10.1016/j.geomorph.2012.10.008 is OK
- 10.1002/esp.1952 is OK
- 10.2110/pec.99.62.0197 is OK
- 10.1016/j.earscirev.2015.10.013 is OK
- 10.1016/j.geomorph.2015.04.007 is OK
- 10.1371/journal.pone.0195557 is OK
- 10.21105/joss.00964 is OK
- 10.1029/2010jf001935 is OK
- 10.1063/1.4823180 is OK
- 10.1016/j.margeo.2011.02.014 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by tristan-salles** [comment]
Hi @kbarnhart,

Thanks for the suggestions!
 
I have merged PR and ran @whedon check references. I made a tagged release of the software, with the following Zenodo reference:
- Tristan Salles, Claire Mallard, & Sabin Zahirovic. (2020, December 13). gospl: Global Scalable Paleo Landscape Evolution (Version 0.2.1). Zenodo. http://doi.org/10.5281/zenodo.4319332

[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4319332.svg)](https://doi.org/10.5281/zenodo.4319332)

**Comment by kbarnhart** [comment]
@whedon generate pdf

**Comment by kbarnhart** [comment]
@whedon set v0.2.1 as version

**Comment by whedon** [comment]
OK. v0.2.1 is the version.

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: :point_left:

**Comment by kbarnhart** [comment]
@whedon set 10.5281/zenodo.4319332 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.4319332" target="_blank">10.5281/zenodo.4319332</a> is the archive.

**Comment by kbarnhart** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1016/j.sedgeo.2018.06.007 is OK
- 10.1029/JB085iB07p03711 is OK
- 10.1029/94JB00744 is OK
- 10.1111/j.1365-2117.1992.tb00136.x is OK
- 10.2110/jsr.2016.56 is OK
- 10.1002/2013WR014326 is OK
- 10.1016/j.cageo.2013.04.024 is OK
- 10.1016/j.epsl.2019.115728 is OK
- 10.5194/gmd-12-4165-2019 is OK
- 10.1046/j.1365-2117.1997.00030.x is OK
- 10.1007/978-1-4615-0575-4_12 is OK
- 10.5194/esurf-5-21-2017 is OK
- 10.5194/esurf-8-379-2020 is OK
- 10.1016/j.geomorph.2012.10.008 is OK
- 10.1002/esp.1952 is OK
- 10.2110/pec.99.62.0197 is OK
- 10.1016/j.earscirev.2015.10.013 is OK
- 10.1016/j.geomorph.2015.04.007 is OK
- 10.1371/journal.pone.0195557 is OK
- 10.21105/joss.00964 is OK
- 10.1029/2010jf001935 is OK
- 10.1063/1.4823180 is OK
- 10.1016/j.margeo.2011.02.014 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1980

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1980, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by kbarnhart** [comment]
@tristan-salles this is now in the hands of the JOSS handling editor in chief who will address final publication steps. 

Thanks @johnjarmitage and @cmshobe for your reviews.

**Comment by tristan-salles** [comment]
Thanks @kbarnhart !

**Comment by kthyng** [comment]
Hi @tristan-salles! I'll be taking over to get your publication finalized. Looks like everything is in place. I have one comment from the paper. They are your own publications, but I see one called "eSCAPE" and one called "ESCAPE" but apparently should both be "eSCAPE". That one extra capital letter gives a really different feeling to the word. Would you like to update this in your references?

**Comment by tristan-salles** [comment]
Hi @kthyng, Thanks for picking this, I have made the changes accordingly.

**Comment by kthyng** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02804/joss.02804/10.21105.joss.02804.pdf) :page_facing_up: :point_left:

**Comment by kthyng** [comment]
Ok looks good!

**Comment by kthyng** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1338583382914109441) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1986
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02804](https://doi.org/10.21105/joss.02804)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by kthyng** [comment]
Congrats on your new publication @tristan-salles! Thank you to editor @kbarnhart and to reviewers @johnjarmitage and @cmshobe for your time and expertise. 🎉 🎉

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02804/status.svg)](https://doi.org/10.21105/joss.02804)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02804">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02804/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02804/status.svg
   :target: https://doi.org/10.21105/joss.02804
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02804/status.svg)](https://doi.org/10.21105/joss.02804)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
